language/vm/vm-runtime/vm-runtime-types/src/native_functions/dispatch.rs

```diff
@@ -243,6 +254,18 @@ lazy_static! {
             vec![ByteArray, U64, TypeParameter(0)],
             vec![]
         );
+        // LibraAccount
+        add!(m, addr, "LibraAccount", "save_account",
+            |_| {
+                Err(VMStatus::new(StatusCode::UNREACHABLE).with_message(
+                    "save_account does not have a native implementation".to_string()))
+            },
+            vec![
+                Address,
+                Struct(StructHandleIndex::new(0), vec![]),
```

Comment:
@tnowacki take a look at this and this is what I was talking to you about... just fyi...

Reply:
Yeah... Its not great, but its not awful. Maybe just a comment at least of what struct it is

Reply:
it was more to give you a sense of what we talked about and the possible other solution

Reply:
Ah, right